packages/notification-center/src/shared/interfaces/index.ts

```diff
@@ -94,16 +94,20 @@ export interface INovuProviderContext {
   logout: VoidFunction;
 }
 
-export interface INotificationsContext {
+export interface IStoreContext {
+  storeQuery: IStoreQuery;
   storeId: string;
   stores: IStore[];
+  setStore: (storeId?: string) => void;
+}
+
+export interface INotificationsContext extends IStoreContext {
   unseenCount: number;
   notifications: IMessage[];
   hasNextPage: boolean;
   isLoading: boolean;
   isFetching: boolean;
   isFetchingNextPage: boolean;
-  setStore: (storeId?: string) => void;
```

Comment:
Need to change the [docs](https://docs.novu.co/notification-center/react/api-reference#usenotifications) for these types :)